ydkgen/printer/meta_data_util.py

```diff
@@ -179,7 +179,7 @@ def format_range_string(ranges):
     return range_string
 
 
-def get_meta_info_data(prop, property_type, type_stmt, identity_subclasses=None):
+def get_meta_info_data(prop, property_type, type_stmt, identity_subclasses=None, language='py'):
```

Comment:
> language='py'
I think language argument should not have a default argument here